<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbConnectionForm</class>
 <widget class="QDialog" name="DbConnectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DB Connection Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/icons.qrc">
    <normaloff>:/icons/server-database.png</normaloff>:/icons/server-database.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Driver</string>
       </property>
       <property name="buddy">
        <cstring>driverCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="driverCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>120</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>QMYSQL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QSQLITE</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="fileNameFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>&amp;FileName</string>
        </property>
        <property name="buddy">
         <cstring>dbFileLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dbFileLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="browseButton">
        <property name="toolTip">
         <string>Browse for existing file</string>
        </property>
        <property name="statusTip">
         <string>Browse for existing file</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="pwFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&amp;Host</string>
        </property>
        <property name="buddy">
         <cstring>hostLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="hostLineEdit">
        <property name="text">
         <string>localhost</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;UserName</string>
        </property>
        <property name="buddy">
         <cstring>userLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="userLineEdit">
        <property name="text">
         <string>amarok</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&amp;Password</string>
        </property>
        <property name="buddy">
         <cstring>passwordLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="text">
         <string>amarok</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="dbLabel">
        <property name="text">
         <string>&amp;Database</string>
        </property>
        <property name="buddy">
         <cstring>databaseLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="databaseLineEdit">
        <property name="text">
         <string>amarok</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../icons/icons.qrc">:/icons/server-database.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>driverCombo</tabstop>
  <tabstop>dbFileLineEdit</tabstop>
  <tabstop>databaseLineEdit</tabstop>
  <tabstop>hostLineEdit</tabstop>
  <tabstop>userLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DbConnectionForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DbConnectionForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
